src/api/c++/z3++.h

```diff
@@ -2935,7 +3046,26 @@ namespace z3 {
         Z3_fixedpoint m_fp;
     public:
         fixedpoint(context& c):object(c) { m_fp = Z3_mk_fixedpoint(c); Z3_fixedpoint_inc_ref(c, m_fp); }
-        ~fixedpoint() { Z3_fixedpoint_dec_ref(ctx(), m_fp); }
+        fixedpoint(fixedpoint const & o) : object{o}, m_fp{o.m_fp} {
+            Z3_fixedpoint_inc_ref(ctx(), m_fp);
+        }
+        fixedpoint(fixedpoint && o) noexcept : object{std::forward<object>(o)}, m_fp{o.m_fp} {
+            o.m_fp = nullptr;
+        }
+        fixedpoint & operator=(fixedpoint const & o) {
```

Comment:
It is definitely a miss that = isn't provided and not declared as del

Reply:
I am failing to read that sentence. What do you mean?

Reply:
The sentence says "it is a bug that fixedpoint object didn't come with operator=.